doc/adr/0002/graphql-server-example/index.js

```diff
@@ -0,0 +1,51 @@
+// https://www.apollographql.com/docs/apollo-server/getting-started/#step-3-define-your-graphql-schema
```

Comment:
Do we need to provide the example code here or could we just have a reference to this example in the GraphQL ADR? It's to demonstrate it with a simple example and it might at some maintenance tasks.

Reply:
Agreed. -Will push an amend, accordingly; thanks.

Reply:
Thanks